SU-HSDO/suhumsci
# STN-626: Hotfix: Card descriptions breaking words unexpectedly

# READY FOR REVIEW

## Summary
We are seeing a regression impacting the way text breaks in postcards, description fields, etc. This PR should address the unexpected text breaks while still allowing very long words to wrap when needed within the container.

Jira Card: [STN-626](https://sparkbox.atlassian.net/browse/STN-626)

Example:
https://creees-prod.stanford.edu/academics/capstone-thesis

## Need Review By (Date)
asap

## Urgency
hight

## Steps to Test
1. npm run test
2. To verify go to any page using hb-cards and verify the `hb-card__description` area of cards is not breaking words at random letters. It should now be only breaking words that are very long and even then it should start a new line with that word before it elects to wrap within the container.
A good place to check is http://sparkbox-sandbox.suhumsci.loc/qa/qa-postcards, inspecting any of the card descriptions should give you a good place to look for proper wrapping within the container.
3. If you edit one of the cards and add a very long word it should have this behavior below.
<img width="936" alt="Screen Shot 2020-11-12 at 3 54 21 PM" src="https://user-images.githubusercontent.com/9355698/99003457-1cec1000-250c-11eb-9a61-f891efc0de2e.png">
<img width="923" alt="Screen Shot 2020-11-12 at 3 55 58 PM" src="https://user-images.githubusercontent.com/9355698/99003442-16f62f00-250c-11eb-949b-b363db40c2b7.png">

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Sparkbox PR Checklist](../docs/SparkboxPRChecklist.md)
